tests/python/PycodestyleBearTest.py

```diff
@@ -22,6 +22,12 @@ def hello():
     valid_files=(good_file,),
     invalid_files=(bad_file,))
 
+PycodestyleBearTest = verify_local_bear(
```

Comment:
You have to name the test differently, otherwise it overrides the test from before and that's not executed then^^